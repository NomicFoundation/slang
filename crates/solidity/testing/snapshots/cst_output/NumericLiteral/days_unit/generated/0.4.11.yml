# This file is generated automatically by infrastructure scripts. Please don't edit by hand.

Source: >
  1  │ 10 days                                                                          │ 0..7

Errors: []

Tree:
  - NumericLiteral (Rule): # 0..7 "10 days"
      - DecimalNumber (Token): "10" # 0..2
      - NumberUnit (Token): "days" # 3..7
