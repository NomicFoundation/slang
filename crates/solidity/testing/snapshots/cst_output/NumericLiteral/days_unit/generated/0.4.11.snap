# This file is generated automatically by infrastructure scripts. Please don't edit by hand.

Errors: 0

█ Rule(NumericLiteral) - (0..7)        │
├──► Token(DecimalNumber) - (0..2)     │ 10
└──► Token(NumberUnit) - (3..7)        │ days
